Apps/UnitTests/Scripts/tests.js

```diff
@@ -101,6 +101,144 @@ describe("RequestFile", function () {
     })
 })
 
+function createSceneAndWait(callback, done) {
+    const engine = new BABYLON.NativeEngine();
+    const scene = new BABYLON.Scene(engine);
+    scene.createDefaultCameraOrLight();
```

Comment:
Do you actually need a light? Are we actually rendering? Also, it doesn't look like camera is used in all cases. Maybe it should be created only in the cases that are needed?

Reply:
True, we don't need a light. For testing materials we don't need a camera, but it didn't seem worth it to duplicate createDefaultCamera() across every PP test case. I guess it would make the individual tests a bit more isolated, but the benefit didn't seem worth it to me.

Reply:
You could pass arguments to the function specifying whether to create them or not. Not a big deal either way.